tests/integration-tranports/src/test/resources/org/mule/test/integration/exceptions/on-error-propagate-use-case-flow.xml

```
@@ -56,21 +56,23 @@
     </flow>
 
     <flow name="fullyDefinedRollbackExceptionStrategyWithComponent">
-        <vm:inbound-endpoint path="in6" exchange-pattern="request-response"/>
+        <vm:inbound-endpoint path="in6" exchange-pattern="request-response">
+            <redelivery-policy maxRedeliveryCount="${shortMaxRedelivery}"/>
+        </vm:inbound-endpoint>
         <append-string-transformer message=" apt1"/>
         <test:component throwException="true"/>
         <error-handler>
-            <on-error-propagate maxRedeliveryAttempts="${shortMaxRedelivery}">
+            <on-error-continue type="REDELIVERY_EXHAUSTED">
```

should this be propagate instead of continue?